.github/workflows/autoformat.yml

```diff
@@ -0,0 +1,46 @@
+name: Code Formatter
+on:
+  pull_request:
+    types:
+      - labeled
+jobs:
+  autoformat:
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'autoformat')
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '^1.14'
+      - name: Install goimports
+        run: go get golang.org/x/tools/cmd/goimports
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          # Specify ref to get a working git environment.
+          ref: ${{ github.event.pull_request.head.ref }}
+      - run: goimports -w .
+      - run: go mod tidy
+      - name: Commit changes
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git config user.name yoshi-automation
+          git config user.email yoshi-automation@google.com
+          git remote set-url origin https://x-access-token:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git
+          git add .
+          if [ -n "$(git status --porcelain)" ]; then
+            git commit --author "yoshi-automation <yoshi-automation@google.com>" --message "Run goimports and go mod tidy"
+            git push
```

Comment:
What ref does this push to?

https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/committing-changes-to-a-pull-request-branch-created-from-a-fork

If it's in someone's fork and they don't have "Allow edits from maintainers" on then this'll fail. Is that working as intended?

https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests

Reply:
Yes, that's intentional. I'm not sure what the failure will look like, exactly, though.